tools/cosmovisor/cmd/cosmovisor/run.go

```diff
@@ -18,17 +18,16 @@ var runCmd = &cobra.Command{
 }
 
 // Run runs the configured program with the given args and monitors it for upgrades.
-func Run(cmd *cobra.Command, args []string, options ...RunOption) error {
+func Run(_ *cobra.Command, args []string, options ...RunOption) error {
```

Comment:
If we remove an argument, let's break the API. There is no consumer of cosmovisor API anyway.

Reply:
makes sense. i'll convert `Run` to private function as well